backend/handler/auth/middleware.py

```diff
@@ -107,7 +108,8 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
                 jwt_claims = self._validate_jwt_payload(jwt_payload)
                 scope["session"] = jwt_claims
                 initial_session_was_empty = False
-            except BadSignatureError:
+
+            except (BadSignatureError, HTTPException):
```

Comment:
i don't think any calls in here will throw an `HTTPException`, do you can probably remove it.

Reply:
You are right!